atemOSC/atemOSC.xcodeproj/project.pbxproj

```diff
@@ -209,7 +551,8 @@
 				LastUpgradeCheck = 0830;
 				TargetAttributes = {
 					8D1107260486CEB800E47090 = {
-						DevelopmentTeam = K5C6E7A2D6;
+						DevelopmentTeam = LRNWZB2D4D;
```

Comment:
This is what breaks code signing

Reply:
Thanks, I did some rebasing and remove unnecessary changes to that file.